### PR TITLE
Examples: Added `hashBlur` and improve `webgpu_backdrop_area` example

### PR DESCRIPTION
**Pull request by sunag**
**Description**

`hashBlur` is more performant in tests, the main reason is because it doesn't need mipmaps, and it seems to have satisfactory quality, although with noise it has a better smoother blur transition amount.

- [x] [added hashBlur](https://github.com/mrdoob/three.js/commit/d849f33395491bdac83119472d2cf035d5992c5d)
- [x] [improve webgpu_backdrop_area example](https://github.com/mrdoob/three.js/commit/1d31f7e8140378d3f260c96e1d72c770ab7db1f2)

image

### CODE REVIEW

**Comment by mrdoob** [comment]
Very nice!

Should transmission use it too?

File: examples/jsm/tsl/display/hashBlur.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { float, Fn, vec2, sin, rand, degrees, cos, Loop, vec4 } from 'three/tsl';
+
+// https://www.shadertoy.com/view/4lXXWn
+
+export const hashBlur = /*#__PURE__*/ Fn( ( [ textureNode, bluramount = float( 0.1 ), repeats = float( 45 ) ] ) => {
+
+	const draw = ( uv ) => textureNode.uv( uv );
+
+	const targetUV = textureNode.uvNode;
```
Comment (Mugen87) [inline examples/jsm/tsl/display/hashBlur.js:9]: There might be texture nodes without `uvNode` so maybe:
```js
const targetUV = textureNode.uvNode || uv();
```

**Comment by sunag** [comment]
https://github.com/mrdoob/three.js/pull/29619

It was necessary 3 fixes, but now it's working.

hashBlur
image

gaussian blur
image

**Comment by mrdoob** [comment]
I noticed `webgpu_ocean` has a similar issue:

https://threejs.org/examples/webgpu_ocean.html

https://github.com/user-attachments/assets/7f4ff67f-5829-4099-be07-76c7d9b99104

**Comment by sunag** [comment]
Fixed too :) https://github.com/mrdoob/three.js/pull/29626

**Pull request merged**